proof/crefine/AARCH64/Finalise_C.thy

```diff
@@ -1403,23 +1403,49 @@ lemma ccorres_ul_pre_getObject_pte:
   apply simp
   done
 
-lemma lookupPTFromLevel_ccorres:
+schematic_goal of_nat_maxPTLevel_val:
+  "of_nat maxPTLevel = (numeral ?n :: machine_word)"
```

Comment:
Are these being schematic_goals a way to avoid magic numbers? If so, is there any reason why `of_nat_ptTranslationBits_NormalPT_T_val` isn't one as well?

Reply:
Most likely reason is that I missed it when I added the `numeral` guards :-)